theme/lumo/vaadin-grid-styles.html

```diff
@@ -88,57 +88,88 @@
+        z-index: 100 !important;
       }
 
-      [part~="row"][row-drag] [part~="cell"] {
+      [part~="row"][dragover] [part~="cell"] {
         overflow: visible;
       }
 
-      [part~="row"][row-drag] [part~="cell"]::before {
+      [part~="row"][dragover] [part~="cell"]::after {
         content: "";
         position: absolute;
-        top: 1px;
+        top: 0;
         right: 0;
-        bottom: 1px;
+        bottom: 0;
         left: 0;
+        height: calc(var(--_lumo-grid-border-width) + 2px);
         pointer-events: none;
+        background: var(--lumo-primary-color-50pct);
       }
 
-      [part~="row"][row-drag="below"] [part~="cell"]::before {
-        box-shadow: 0 3px 0 0 var(--lumo-primary-color-50pct);
+      :host([theme~="no-row-borders"]) [dragover] [part~="cell"]::after {
+        height: 2px;
       }
 
-      [part~="row"][row-drag="above"] [part~="cell"]::before {
-        box-shadow: 0 -3px 0 0 var(--lumo-primary-color-50pct);
+      [part~="row"][dragover="below"] [part~="cell"]::after {
+        top: 100%;
+        bottom: auto;
+        margin-top: -1px;
       }
 
-      [part~="row"][row-drag="on-top"] [part~="cell"]::before {
-        background: var(--lumo-primary-color-50pct);
+      [part~="row"][dragover="above"] [part~="cell"]::after {
+        top: auto;
+        bottom: 100%;
+        margin-bottom: -1px;
+      }
+
+      [part~="row"][dragover][dragover="on-top"] [part~="cell"]::after {
+        height: 100%;
+      }
+
+      [part~="row"][dragstart] {
+        /* Add bottom-space to the row so the drag number doesn't get clipped. Needed for IE/Edge */
+        border-bottom: 100px solid transparent;
+        z-index: 100 !important;
+        opacity: 0.9;
+      }
+
+      [part~="row"][dragstart] [part~="cell"] {
+        border: none !important;
+        box-shadow: none !important;
+      }
+
+      [part~="row"][dragstart] [part~="cell"][last-column] {
+        border-radius: 0 var(--lumo-border-radius-s) var(--lumo-border-radius-s) 0;
+      }
+
+      [part~="row"][dragstart] [part~="cell"][first-column] {
+        border-radius: var(--lumo-border-radius-s) 0 0 var(--lumo-border-radius-s);
       }
 
-      [part~="row"][dragged-row-count]::before {
+      [part~="row"][dragstart]:not([dragstart=""])::after {
         display: block;
         position: absolute;
-        bottom: 0;
-        left: 30%;
+        left: var(--_grid-drag-start-x);
+        top: var(--_grid-drag-start-y);
         z-index: 100;
-        content: attr(dragged-row-count);
+        content: attr(dragstart);
         align-items: center;
         justify-content: center;
         box-sizing: border-box;
-        padding: 0.4em calc(0.5em + var(--lumo-border-radius-s) / 4);
-        color: var(--lumo-primary-contrast-color);
-        background-color: var(--lumo-primary-color);
-        border-radius: var(--lumo-border-radius-s);
+        padding: 0.3em calc(0.3em + var(--lumo-border-radius-s) / 4);
```

Comment:
**Suggestion:** `--lumo-space-xs` or `--lumo-space-xs * 1.2`.

Reply:
Might be better to use `--lumo-space-xs` indeed. This pattern is copied & adjusted from https://github.com/vaadin/vaadin-lumo-styles/blob/master/badge.html#L13

Reply:
Done.